e2e/cypress/integration/ui/settings/project_mgmt.spec.ts

```diff
@@ -75,171 +77,190 @@ describe('project management', () => {
+    cy.get('app-notification.info')
+      .contains(`Created project ${customWithPolsProjectID} and associated policies`);
+    cy.get('app-notification.info chef-icon').click();
+
+    cy.contains(customWithPolsProjectID).should('exist');
+
+    // verify associated policies were generated
+    associatedPolicies.forEach((policy) => {
+      cy.request({
+        auth: { bearer: adminIdToken },
+        url: `/apis/iam/v2/policies/${customWithPolsProjectID}-${policy}`
+      }).then((response) => {
+        expect(response.status).to.equal(200);
+      });
+    });
 
     cy.url().should('include', '/settings/projects');
-    cy.get('app-project-details chef-td').contains(updatedRuleName);
-    cy.get('app-project-details chef-td').contains('2 conditions');
-    cy.get('app-project-details').contains('Edits are pending');
   });
 
-  itFlaky('can update a project name', () => {
+  it('can open the new project\'s details page', () => {
+    cy.get('[data-cy=project-details]').contains(projectName).click();
+    cy.url().should('include', `/settings/projects/${projectID}`);
+  });
+
+  it('can update a project name', () => {
     const updatedProjectName = `updated ${projectName}`;
     cy.route('GET', `/apis/iam/v2/projects/${projectID}`).as('getProject');
 
     cy.get('[data-cy=details-tab]').click();
     cy.wait('@getProject');
 
     cy.get('[data-cy=update-project-name]').focus().clear()
-      .type(updatedProjectName, { delay: typeDelay })
+      .type(updatedProjectName)
       .should('have.value', updatedProjectName);
     cy.get('app-project-details chef-button').contains('Save').click();
 
     cy.get('app-project-details h1.page-title').contains(updatedProjectName);
   });
 
-  itFlaky('can delete a project rule', () => {
-    cy.get('app-project-details').contains('Edits are pending');
-    cy.get('[data-cy=rules-tab]').click();
+  it('cannot delete a project with rules', () => {
+    const rule: Rule = {
+      id: `${cypressPrefix}-rule`,
+      name: 'test rule',
+      project_id: customWithoutPolsProjectID,
+      type: 'NODE',
+      conditions: [
+        {
+          attribute: 'CHEF_ORGANIZATION',
+          operator: 'EQUALS',
+          values: ['foo']
+        }
+      ]
+    };
+    // add rule via the API to save time,
+    // since we test rules more thoroughly in project_rule_mgmt.spec.ts
+    cy.request({
+      auth: { bearer: adminIdToken },
+      method: 'POST',
+      url: `/apis/iam/v2/projects/${customWithoutPolsProjectID}/rules`,
+      body: rule
+    }).then((response) => {
+      expect(response.status).to.equal(200);
+    });
+
+    // navigate back to the project list page and get latest project data
+    cy.route('GET', '/apis/iam/v2/projects').as('getProjects');
+    cy.get('.breadcrumb').click();
+    cy.wait('@getProjects');
+
+    cy.get('app-project-list chef-td').contains(customWithoutPolsProjectID).parent()
+      .find('.mat-select-trigger').as('controlMenu');
```

Comment:
I always had trouble selecting the right element for the control menu in these tests and getting a successful interaction, but it looks like our `mat-select` refactor made it easier to target :)